src/lwc/utilPlatformEventListener/utilPlatformEventListener.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Created by kenneth.lewis on 2019-06-27.
+ */
+
+import {LightningElement, track, api} from 'lwc';
+import { subscribe, unsubscribe, onError, setDebugFlag, isEmpEnabled } from 'lightning/empApi';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import { fireEvent } from 'c/pubsubNoPageRef';
+
+export default class PlatformEventListener extends LightningElement {
+    @api channelName = '/event/DeploymentEvent__e'; //Default
```

Comment:
Is this going to need a namespace when the event is in a managed package but the org doesn't have a default namespace?